### ISSUE TITLE
R error causes add trial design to fail on spbase-test

### ISSUE THREAD

**Issue by bellerbrock** [solgenomics/sgn#4146]
Expected Behavior 
--------------------------------------------------------------------------

![rerror](https://user-images.githubusercontent.com/11297346/175331479-c8a169dc-a2ba-4935-9a81-66b5ea304618.png)

Related to the upgrade to debian 11?

For Bugs:
---------

**Comment by titima15** [comment]
@bellerbrock After upgraded to debian 11, R_libs has to be on this branch:topic/debian_bullseye_R4.1.2

**Comment by bellerbrock** [comment]
Thanks @titima15. I've updated sweetpotatobase-test now to the latest sgn tag, and put R_libs on the `topic/debian_bullseye_R4.1.2` branch. The problem persists though. Here are more details from the error log:
```
DESIGN TYPE = RCBD
Error: package or namespace load failed for 'agricolae' in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/production/cxgn/R_libs/vctrs/libs/vctrs.so':
  /home/production/cxgn/R_libs/vctrs/libs/vctrs.so: undefined symbol: EXTPTR_PROT
In addition: Warning message:
package 'agricolae' was built under R version 4.1.2 
Execution halted
```

**Comment by isaak** [comment]
@bellerbrock, the R version in spbase-test is old (4.0). Try to upgrade it to the latest R (4.2).  The R packages in the branch are compiled against a more recent R and that may be causing the problem (for some packages). I tried to create a trial on my local machine, R 4.2 and the same branch, and succeeded.

Here is some helpful instruction on upgrading R: https://www.how2shout.com/linux/install-the-latest-r-programming-language-version-on-debian-11-bullseye/

From the 'Add R Repository on Debian 11 Bullseye' part...

**Comment by dwaring87** [comment]
I've been testing some of the trial designs on our instances and the Randomized and RCBD designs work on Debian 11 with R 4.2.0 using the R_libs branch `topic/debian_bullseye_R4.1.2` (we don't have Isaak's latest updates in that branch yet).

**Comment by bellerbrock** [comment]
Thanks @isaak and @dwaring87! Updating the sweetpotatobase-test.sgn.cornell.edu server R version to 4.2 did the trick. I made the mistake of not keeping the locally modified configuration file during the upgrade, so also had to add `R_LIBS_USER=${R_LIBS_USER-'/home/production/cxgn/R_libs'}` back to the `/etc/R/Renviron` file before it all worked.

I also tested the Resolvable Row-Column design option which submits a job to run on the cluster, that seems to be working again too.
